src/main/resources/com/metype/game/workingTitleScreen.fxml

```diff
@@ -66,7 +66,7 @@
          <Insets bottom="10.0"/>
       </HBox.margin>
       <Button fx:id="storyButton" maxHeight="1000.0" maxWidth="1000.0" mnemonicParsing="false"
-              onAction="#storyButtonClick" prefHeight="60.0" prefWidth="213.0" text="Story" textAlignment="CENTER"
```

Comment:
I would prefer to leave a link to the handler as a story is planned in the near future, though I do understand the reasoning behind removing it for now.